snuba/datasets/storages/transactions.py

```diff
@@ -148,6 +149,7 @@
                 "title",
             ]
         ),
+        ProjectReferrerRateLimiter("project_id"),
```

Comment:
Help: mypy gives me a type error here and I have no idea why

Is this not a query processor?

```
snuba/datasets/storages/transactions.py:152: error: List item 13 has incompatible type "ProjectReferrerRateLimiter"; expected "QueryProcessor"
```